### DISCUSSION TITLE
Where i need to send early proposal for review ?

### DISCUSSION THREAD

**Discussion by Aman123lug** [openvinotoolkit/openvino#16535] [Google Summer of Code]
My self Aman Kumar. I write a proposal for a project I want to feedback from a mentor where can I send my proposal?

**Comment by adrianboguszewski** [comment]
What project are we talking about?

**Reply by Aman123lug** [reply]
1. Smart meeting notes
2. Add OpenVINO support to John Snow Labs Spark NLP 
I have very good experience in Machine learning and deep learning and I created many big projects in data science my experience is enough for completing this project I question if any PR is required in order to select for Gsoc 23

**Reply by adrianboguszewski** [reply]
1. @eaidova @andrei-kochin 
2. @junwenwu @ravi9 

Guys, please advice

**Reply by Aman123lug** [reply]
no updates?

**Reply by eaidova** [reply]
@Aman123lug do you provided PR with the prerequisite task https://github.com/openvinotoolkit/openvino_notebooks/issues/832?

**Reply by ravi9** [reply]
@Aman123lug Sorry for the delayed response. Were you able to assign one of the prerequisite task from [here](https://github.com/openvinotoolkit/openvino_notebooks/issues/832) ? 
After you finish the prerequisite task, you could share the proposal our email address: ravi.panchumarthy@intel.com and junwen.wu@intel.com

Thanks,
Ravi.
